ui/src/components/side_menu.jsx

```diff
@@ -1,107 +0,0 @@
-import React from 'react';
```

Comment:
This was leftover from an earlier prototype and we're not using it anymore so ✂️